xml/entity-decl.ent

```diff
@@ -7,6 +7,8 @@
 <!ENTITY sle "&suse; Linux Enterprise">
 <!ENTITY ses "&suse; Linux Enterprise">
```

Comment:
This is definitely wrong. it needs to be:
```
suggestion
<!ENTITY ses "&suse; Enterprise Storage">
```
But I suggest you remove this (and the next) entity, it's not used in the style guide.